articles/appliance/cli/backing-up-the-appliance.md

```diff
@@ -5,46 +5,62 @@ description: How to back up the Appliance using its CLI
 
 # How to Back Up Appliance Instances Using the CLI
 
-You may use Appliance CLI to perform a backup on a specific node.
+You may use Appliance CLI to perform a Mongo backup on a specific node.
+
+Beginning with version `11638`, the backup doesn't include sensitive configuration information such as encryption keys.
 
 ## Prior to Beginning the Backup
 
-Please ensure that your workstation has been [configured to perform operations](/appliance/cli/configure-cli) on your Appliance instances;
+Please ensure that:
+* You have configured the [Command Line Interface](/appliance/cli/configure-cli) on your Appliance instances;
+* The node has disk space equal to or greater than twice the amount of Auth0 data present.
 
-::: note
-  Please perform backups only on Appliance instances running **build 7247** or later. Backups are only allowed on nodes added to the `backup` role. Please see [Adding a Node to the Backup Role](/appliance/cli/adding-node-to-backup-role) for additional information.
+::: panel Backup Role
```

Comment:
Revert back to note? Use always notes or warnings, unless you have a block of text (4+ lines).